deploy/helm-chart/values.yaml

```diff
@@ -0,0 +1,51 @@
+nginxGateway:
+  ## A GatewayClass resource with the name equal to the class must be deployed. Otherwise, NGINX Kubernetes Gateway
```

Comment:
this reads like the user needs to deploy the GatewayClass resource, rather than the customization for the name.